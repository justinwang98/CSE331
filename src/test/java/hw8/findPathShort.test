# Verify that the program finds and correctly outputs the shortest path from
# BAG to BAG (NE).
r
BAG
BAG (NE)

# All of your test scripts should end with q to terminate the program.
q