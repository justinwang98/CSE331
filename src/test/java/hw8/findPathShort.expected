Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Verify that the program finds and correctly outputs the shortest path from
# CSE to PAB.
Abbreviated name of starting building: Abbreviated name of ending building: Path from Bagley Hall (East Entrance) to Bagley Hall (Northeast Entrance):
	Walk 21 feet NE to (1923, 1705)
	Walk 148 feet NW to (1894, 1641)
	Walk 47 feet SW to (1873, 1650)
	Walk 25 feet SE to (1878, 1661)
Total distance: 241 feet

Enter an option ('m' to see the menu): 
# All of your test scripts should end with q to terminate the program.
